### ISSUE TITLE
test failed - upload document

### ISSUE THREAD

**Issue by ValuedMammal** [Avimitin/deepl-rs#17]
If a translation changes subtly on the server side, then asserting the exact content in a unit test will fail. Asserting the exact translation of a string like 'Hello world' is probably harmless, but since we don't control the result of translation, we might come up with another way to test library functions.

```
thread 'endpoint::document::test_upload_document' panicked at 'assertion failed: `(left == right)`
  left: `"怀疑星星是火。怀疑太阳在移动。怀疑真理是骗子。但永远不要怀疑我的爱"`,
 right: `"怀疑你的星星是火。怀疑太阳在动。怀疑真理是个骗子。但永远不要怀疑我的爱。"`', src/endpoint/document.rs:362:5
```

A quick fix would be to change the expected content string, but I had some other ideas for making tests more robust:

- Assert the translation `contains` only a subset of the expected content.
- Assert the translated string is at least an expected `len`.
- Read the translated file content to a string and send it back to the translator, then assert the detected source lang of the second translation, in this case `Lang::ZH` - but this would also introduce an extra request.

Curious if you have any opinions

**Comment by Avimitin** [comment]
I am worried that using contain or asserting length of the translated text are also hacky... And DeepL doesn't promised that the content between two language is steady, so maybe we will get new content after sending translated text back to server.

Emmmm, maybe we should use LLM to help us check the similarity between translated text and the expected text? JK, but I have no idea for a simple and lightweight approach. Maybe shorten the testing text will be a better way.

**Comment by ValuedMammal** [comment]
perhaps it's sufficient to test merely that a file exists at the given download path (provided it wasn't there before).

**Comment by Avimitin** [comment]
Should be fixed in dc988f3d37678417a599894b77ba8407a06c44d1. Also change the target language to Deutsch. English and Deutsch are born from same language family, converting between them can reduce the semantic diversity.
